plugins/modules/inventory_intent.py

```diff
@@ -1195,34 +1195,44 @@ def export_device_details(self):
                 self.result['response'] = self.msg
                 return self
 
-            payload_params = {
-                "deviceUuids": device_uuids,
-                "password": password,
-                "operationEnum": export_device_list.get("operation_enum", "0"),
-                "parameters": export_device_list.get("parameters")
-            }
-
-            response = self.trigger_export_api(payload_params)
-            self.check_return_status()
+            # Export the device data in a batch of 500 devices at a time
+            start = 0
+            chunk_size = 500
```

Comment:
May we pass this value from the playbook variable? Let's assume that the default value is 500 if users do not pass this value from the playbook. In the documentation, we can mention that this API currently supports a maximum of 500 devices. If any error occurs, then the value should be reduced.